rust/src/chrome.rs

```diff
@@ -156,31 +164,29 @@ impl SeleniumManager for ChromeManager {
             _ => {
                 let mut driver_version = "".to_string();
                 let mut browser_version_int = browser_version.parse::<i32>().unwrap_or_default();
-                if browser_version_int > 0 {
```

Comment:
Why is this `if` removed?

Reply:
I have just committed a new patch to consider the original motivation of that condition, i.e., to avoid checking negative driver versions (although it is basically an error case that happens when a user forces a very low Chrome version, like `2`).

Reply:
👍